### PR TITLE
fix(#726): Icon component should have ability to remove title tab inside svg

### PR DESCRIPTION
**Pull request by xiaolixu**
Closes IBM/carbon-components-react#726

Icon component should have ability to remove title tab inside svg

#### Changelog

**Changed**
**Icon**

* add _iconTitle_ prop , and the Icon's title will still use _description_ prop if  _iconTitle_ is _undefined_ 

**Tooltiop**

* add _iconTitle_ prop , and the default value is _""_

### CODE REVIEW

**Comment by tw15egan** [comment]
Since `aria-label` is there, I'm fine with this change as this should not affect a11y

File: src/components/Tooltip/Tooltip.js [hunk 1]
```diff
@@ -171,10 +171,15 @@ export default class Tooltip extends Component {
     iconName: PropTypes.string,
 
     /**
-     * The description of the default tooltip icon, to be put in its SVG `<title>` element.
+     * The description of the default tooltip icon, to be put in its SVG `aria-lable` and 'alt' .
```
Comment (tw15egan) [inline src/components/Tooltip/Tooltip.js]: small typo, should be `aria-label`

**Comment by joshblack** [comment]
Bump @xiaolixu for status 👍

**Comment by xiaolixu** [comment]
@joshblack  The root cause of the linked issue is that the parameters "description" "alt" and "title" in the component "Icon" use the same variable, so iconTitle is added for parameter "title".

**Comment by asudoh** [comment]
@xiaolixu Thank you for jumping in and my apologies for being late. The code change looks good from my perspective. Wrt Storybook, do you want to change `Icon-story.js`? One way to do this is something like this (though I haven't tested it):

```diff
--- a/src/components/Icon/Icon-story.js
+++ b/src/components/Icon/Icon-story.js
@@ -42,9 +42,10 @@ const props = () => {
     width: text('The SVG `width` attribute (width)', ''),
     height: text('The SVG `height` attribute (height)', ''),
     description: text(
-      'The content in <title> in SVG (description)',
+      'The a11y text (description)',
       'This is a description of the icon and what it does in context'
     ),
+    iconTitle: text('The content in <title> in SVG (iconTitle)', ''),
     className: 'extra-class',
   };
 };
```

**Comment by alisonjoseph** [comment]
@xiaolixu did you want to make the storbook updates Akira suggested so we can get this merged in?

**Comment by joshblack** [comment]
Going to close this due to inactivity. Feel free to re-open the PR if you're still willing to take this on! Just a couple of notes left around Storybook and comments updates 👍

**Comment by xiaolixu** [comment]
@asudoh @alisonjoseph   Sorry for no response much time.  I took one long holiday and missed your info after i am back.  now  open one new pr to update the storybook , thanks!

**Pull request closed without merging**